.github/workflows/ci.yml

```diff
@@ -32,9 +32,11 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: yarn
       - name: Install dependencies and build sequelize
-        run: yarn install --frozen-lockfile
+        run: yarn workspace @sequelize/core install --frozen-lockfile
+      - name: Build sequelize
+        run: yarn workspace @sequelize/core build
       - name: Compress artifact
-        run: tar -cf install-build-node-${{ matrix.node-version }}.tar ./lib ./node_modules ./types
+        run: tar -cf install-build-node-${{ matrix.node-version }}.tar ./packages/core/lib ./packages/core/node_modules ./packages/core/types ./node_modules
```

Comment:
Which dependencies are in `./node_modules`?

Reply:
Considering that for the `typedoc` check we still install the root dependencies

Reply:
Yarn workspace dedupes dependencies needed by the different packages of the workspace by putting them in the root node_modules

Got to remove the redundant install from the docs script

Reply:
Done